webauthn/public-key-credential-creation-options-from-json.https.window.js

```diff
@@ -45,7 +46,7 @@ test(() => {
     ],
     // The spec defaults the following fields:
     attestation: "none",
-    hints: [],
+    hints: ["duplicated-value", "duplicated-value"]
```

Comment:
This change means we no longer test the default for the `hints` field. If we think there's value in testing this, we should have a separate `parseCreationOptionsFromJSON` call with the duplicate values instead.